projects/core/src/model/product.model.ts

```diff
@@ -209,6 +209,7 @@ export enum VariantQualifier {
   COLOR = 'color',
   THUMBNAIL = 'thumbnail',
   PRODUCT = 'product',
+  SWATCH = 'styleSwatch',
```

Comment:
[MEDIUM]
For consistency and easier discoverability, let's not shorten original values in the key name
```suggestion
  STYLE_SWATCH = 'styleSwatch',
```

Reply:
fixed